spec.md

```diff
@@ -592,6 +594,204 @@ After implementing the first endpoint `GET /v2/catalog` documented
 [above](#catalog-management), the Service Broker will need to be registered
 with your Platform to make your services and plans available to end users.
 
+## Service Dependencies
+
+Broker authors can define dependencies of their services to other services.
+Dependencies are declared in the catalog at the plan level.
+There are three types of dependencies.
+
+* Open dependencies are arbitrary dependencies.
+  The service instance accepts any bindings and service instance IDs and
+  evaluates them at runtime.
+* Service dependencies are specified by a service or a set of plans.
+  When the service instance is provisioned, the platform has to provide a
+  binding or a service instance ID of a matching service instance.
+* Instance dependencies declare dependencies to a specific instance either
+  by a service instance ID or by a service instance alias.
+
+Dependencies are injected as parameters when a service instance is provisioned.
+The names of the parameters are declared in the catalog.

```

Comment:
does this mean the field that the broker will inject the data that was in the catalog is configurable? is that needed?